draw_samples.py

```diff
@@ -276,15 +259,19 @@ def main():
         subprocess.run(shlex.split(f"mkdir -p {save_init_dir}"))
 
     # Load model
-    if args.type == "backbone":
-        if args.model_checkpoint:
-            checkpoint = f"{args.model_checkpoint}/backbone_state_dict.pth"
-            cfg_path = f"{args.model_checkpoint}/backbone_pretrained.yml"
-        else:
-            checkpoint = (
-                f"{model_directory}/checkpoints/epoch{epoch}_training_state.pth"
-            )
-            cfg_path = f"{model_directory}/configs/backbone.yml"
+    if args.type == "backbone_old":
```

Comment:
Do we intend to support both backbone_old and backbone_new publicly, at the same time? if not, perhaps we should keep backbone_old (inc. the weights) out of this PR and just for internal testing.

Reply:
It was actually Gina's recommendation. I'll commit another one right now.